activestorage/app/models/active_storage/blob.rb

```diff
@@ -201,8 +201,8 @@ def representable?
   # with users. Instead, the +service_url+ should only be exposed as a redirect from a stable, possibly authenticated URL.
   # Hiding the +service_url+ behind a redirect also gives you the power to change services without updating all URLs. And
   # it allows permanent URLs that redirect to the +service_url+ to be cached in the view.
-  def service_url(expires_in: service.url_expires_in, disposition: "inline")
-    service.url key, expires_in: expires_in, disposition: disposition, filename: filename, content_type: content_type
+  def service_url(expires_in: service.url_expires_in, disposition: :inline)
```

Comment:
I've changed this from `"inline"` to `:inline` because I've seen it used as symbol in other places like:
https://github.com/rails/rails/blob/5a5014688873f1d6e1b66075eea8a4356b5a4d07/activestorage/app/models/active_storage/variant.rb#L70-L72

https://github.com/rails/rails/blob/5a5014688873f1d6e1b66075eea8a4356b5a4d07/activestorage/lib/active_storage/service.rb#L91-L93